lightning/src/util/enforcing_trait_impls.rs

```diff
@@ -9,7 +9,7 @@ use bitcoin::blockdata::transaction::Transaction;
 use bitcoin::blockdata::script::Script;
 
 use secp256k1;
-use secp256k1::key::{SecretKey, PublicKey};
+use secp256k1::key::{SecretKey,PublicKey};
```

Comment:
nit: useless diff.

Reply:
reverted this change (not sure how it happened, since IntelliJ puts in spaces), and also changed same in `chan_utils.rs`